### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jml**
Alexis asked us to update the README to

- Remove the availability warning
- Focus on telling people how to build
- Direct people to the Weave Cloud website for more information
- Move language away from replacement Prometheus and toward how it relates to Prometheus community/ecosystem

This PR makes those changes. 

Anita, I welcome feedback from you, especially wrt how this relates to the rest of our documentation architecture.

Aaron, I'm particularly keen to make sure that this ties in with the onboarding work you're doing. i.e. that people who come to this GitHub page, think "I want to use Cortex with Weave Cloud", and follow the cloud.weave.works link will be given enough information to actually begin using cortex on Weave Cloud.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,12 +45,19 @@ To update dependencies, run
 
     dep ensure --update && dep prune
 
+## Further reading
+
+To learn more about Cortex, consult the follow documents / talks:
```
Comment (rade) [inline README.md]: follow -> following

File: README.md [hunk 2]
```diff
@@ -5,69 +5,11 @@
 [![Circle CI](https://circleci.com/gh/weaveworks/cortex/tree/master.svg?style=shield)](https://circleci.com/gh/weaveworks/cortex/tree/master)
 [![Slack Status](https://slack.weave.works/badge.svg)](https://slack.weave.works)
 
-*NB this is a pre-beta service. Availability is not 100%. APIs might change. Data could be lost.*
-
-Cortex is an API compatible [Prometheus](https://prometheus.io) implementation, that natively supports multitenancy and horizontal scale-out clustering.
-
 Cortex is a [Weaveworks](https://weave.works) project that forms the monitoring backend of [Weave Cloud](https://cloud.weave.works).
 
+It provides horizontally scalable, long term storage for [Prometheus](https://prometheus.io) metrics when used as a [remote write](https://prometheus.io/docs/operating/configuration/#remote_write) destination.
```
Comment (rade) [inline README.md]: Reading this makes me wonder how I'd get the data out of it again; I think it's important to mention that Cortex also provides a horizontally scalable, Prom-compatible query API.

File: README.md [hunk 3]
```diff
@@ -5,69 +5,11 @@
...
-
-To use the Prometheus Service with Grafana, configure your Grafana instance to have a datasource with the following parameters:
-
-- Type: Prometheus
-- Url: https://cloud.weave.works/api/prom
-- Access: Proxy
-- Http Auth: Basic Auth
-- User: `user` (this is ignored, but must be non-empty)
-- Password: <Service Token> (the same one your retrieval agent uses)
+If you would like to use Cortex, we recommend signing up for [Weave Cloud](https://cloud.weave.works) and following the instructions there.
```
Comment (abuehrle) [inline README.md]: Additional help can also be found in the [Weave Cloud documentation](https://www.weave.works/docs/cloud/latest/overview/): 

* [Installing the Weave Cloud Agents](https://www.weave.works/docs/cloud/latest/install/installing-agents/#weave-cloud-supported)
* [Prometheus Monitoring in Weave Cloud](https://www.weave.works/docs/cloud/latest/concepts/prometheus-monitoring/)
* [Instrumenting Your App: Best Practises](https://www.weave.works/docs/cloud/latest/concepts/prometheus-monitoring/)

File: README.md [hunk 4]
```diff
@@ -5,69 +5,11 @@
...
```
Comment (abuehrle) [inline README.md]: To use Cortex, sign up for [Weave Cloud](https://cloud.weave.works) and follow the instructions there.

File: README.md [hunk 5]
```diff
@@ -103,12 +45,19 @@ To update dependencies, run
 
     dep ensure --update && dep prune
 
+## Further reading
+
+To learn more about Cortex, consult the following documents / talks:
+
+- [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
```
Comment (abuehrle) [inline README.md]: Should we be linking to design docs here?  Personally, I wouldn't.

File: README.md [hunk 6]
```diff
@@ -103,12 +45,19 @@ To update dependencies, run
...
+
 ## <a name="help"></a>Getting Help
 
-If you have any questions regarding Cortex our hosted Prometheus as a service:
+If you have any questions regarding Cortex, or on using Prometheus with Weave Cloud:
 
-- Invite yourself to the <a href="https://weaveworks.github.io/community-slack/" target="_blank"> #weave-community </a> slack channel.
-- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> slack channel.
+- Join us on [our Slack channel](https://slack.weave.works) and ask us directly!
 - Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
```
Comment (abuehrle) [inline README.md]: That's not the correct slack channel. It should be the one listed on the website, which is what was there before, i.e.Invite yourself to the <a href="https://weaveworks.github.io/community-slack/" target="_blank"> #weave-community </a> slack channel.
 -- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> slack channel.

File: README.md (hunk 5 above)
Comment (jml) [inline README.md]: I'd like to keep it. This is a GitHub repo and we can presume the audience is technical. The presence of a design doc is a good signal for many, even if they don't read it.

File: README.md (hunk 6 above)
Comment (jml) [inline README.md]: Thanks. There were two different URLs in the original version.

I've taken the liberty of capitalizing "Slack".

**Comment by jml** [comment]
Thanks for the feedback @abuehrle. Updates applied. PTAL.

File: README.md (hunk 6 above)
Comment (abuehrle) [inline README.md]: If they are not already a member they have to sign up first, which is a different URL. So you need to add this bit as well: 

To invite yourself to the Weave Community Slack channel, visit https://weaveworks.github.io/community-slack/.

File: README.md [hunk 7]
```diff
@@ -57,7 +63,7 @@ To learn more about Cortex, consult the following documents / talks:
 
 If you have any questions regarding Cortex, or on using Prometheus with Weave Cloud:
 
-- Join us on [our Slack channel](https://slack.weave.works) and ask us directly!
+- Ask a question on the <a href="https://weave-community.slack.com/messages/general/"> #weave-community</a> Slack channel
 - Join the <a href="https://www.meetup.com/pro/Weave/"> Weave User Group </a> and get invited to online talks, hands-on training and meetups in your area.
```
Comment (abuehrle) [inline README.md]: other than that LGTM

File: README.md (hunk 5 above)
Comment (abuehrle) [inline README.md]: 👍

**Pull request merged**